.swiftpm/xcode/xcshareddata/xcschemes/BlueprintUI-Package.xcscheme

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```

Comment:
I committed these schemes to work around a bug in Xcode 14. More details linked from the comment in the gitignore file.